manifests/templates/registry-file-host.yaml.in

```diff
@@ -0,0 +1,121 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata: 
+        creationTimestamp: null
+        labels:
+                io.kompose.service: lb-claim0
+        name: lb-claim0
+        namespace: {{ .Namespace }}
+spec:
+        accessModes:
+                - ReadWriteOnce
+        resources:
+                requests:
+                        storage: 100Mi
+status: {}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cdi-func-test-registry-host
+  namespace: {{ .Namespace }}
+  labels:
+    cdi.kubevirt.io: ""
+    cdi.kubevirt.io/testing: ""
+spec:
+  selector:
+    matchLabels:
+      cdi.kubevirt.io/testing: ""
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        cdi.kubevirt.io/testing: ""
+    spec:
+     initContainers:
+     - name: init
+       image: {{ .DockerRepo }}/cdi-func-test-file-host-init:{{ .DockerTag }}
+       imagePullPolicy: {{ .PullPolicy }}
+       args: ["-inFile", "/tmp/source/tinyCore.iso", "-outDir", "/tmp/shared/images"]
+       volumeMounts:
+       - name: "images"
+         mountPath: "/tmp/shared"
+     containers:
+     - name: cdi-func-test-docker-registry
```

Comment:
should be docker-registry (same naming convention as other containers)

Reply:
sure